### ISSUE TITLE
Work with High Sierra?

### ISSUE THREAD

**Issue by fruitofgeorge** [kfix/SleepProxyServer#7]
I installed this on a Raspberry Pi on my network along with two macs running High Sierra. It shows up and looks like it's doing something but I'm still unable to wake one mac from the other using ssh or screen sharing. Should this still theoretically work?

**Comment by kfix** [comment]
yes, but I didn't lock dependencies on it when it was last known to work (Yosemite) so its hard to say
